README.md

```diff
@@ -554,6 +554,8 @@ These are items relevant *only* on OS X.
 
 - You can perform and script most Windows system administration tasks from the command line by learning and using `wmic`.
 
+- Another option to get Unix look and feel under Windows is [Cash](https://github.com/dthree/cash).
```

Comment:
The *cash* idea is very interesting! A disclaimer needs to be added regarding the available functionality; something like: "Note that only very few Unix commands and command-line options are available in this environment".

While at it consider also adding [MinGW](http://www.mingw.org/) with an corresponding disclaimer, and a few bullets guiding people to select each of the three options.  The *Cygwin* bullet should then be merged into this section.